README.md

```diff
@@ -258,6 +261,25 @@ Nov 08, 01:30 +03:00 – skip
 Nov 09, 01:30 +03:00 – run
 ```
 
+<a name="jitter"></a>
+## Jitter
+
+Cronos supports the ability to distribute cron fields randomly in order to spread out system load over time, a feature called "schedule jitter". You can opt into this capability by passing in a pre-seeded random number generator (RNG) and optionally using the special character `H` in a cron expression. Using `H` in an expression while failing to provide an RNG will throw an exception.
```

Comment:
Thanks a lot for the documentation! It can also be worth adding some API example on how to parse and use this feature, since another overload is expected to be used here.